android/src/ti/map/TiUIMapView.java

```diff
@@ -865,6 +865,10 @@ public void onMapClick(LatLng point) {
 		}
 		clickablePolylines.clear();
 
+		KrollDict d = new KrollDict();
+		d.put(TiC.PROPERTY_LATITUDE, point.latitude);
+		d.put(TiC.PROPERTY_LONGITUDE, point.longitude);
+		proxy.fireEvent("mapclick", d);
```

Comment:
this is the whole output: https://gist.github.com/m1ga/2a5ff0409d295f4eea84155df3e307c4